multiqc/utils/search_patterns.yaml

```diff
@@ -264,7 +264,7 @@ dragen/wgs_contig_mean_cov:
 dragen/wgs_coverage_metrics:
   fn_re: '.*\.wgs_coverage_metrics_?(tumor|normal)?\.csv'
 dragen/qc_region_coverage_metrics:
-  fn_re: '.*\.qc-coverage-region-(1|2|3)_coverage_metrics.csv'
+  fn_re: '.*\.qc-coverage-region-([^_]+)_coverage_metrics.csv'
```

Comment:
DRAGEN supports up to 5 regions. A region can be named by a tag via the command line argument so it's not always 1-5.